operations/app/src/modules/event_hub/main.tf

```diff
@@ -0,0 +1,53 @@
+terraform {
+    required_version = ">= 0.14"
+}
+
+resource "azurerm_eventhub_namespace" "eventhub_namespace" {
+  name = "${var.resource_prefix}-eventhub"
+  location = var.location
+  resource_group_name = var.resource_group
+  sku = "Standard"
+  capacity = 1
+
+  tags = {
+    environment = var.environment
+  }
+}
+
+resource "azurerm_eventhub" "frontdoor_access" {
+  name = "${var.resource_prefix}-front_door-access-log"
+  namespace_name = azurerm_eventhub_namespace.eventhub_namespace.name
+  resource_group_name = var.resource_group
+  partition_count = 1
+  message_retention = 1
+}
+
+resource "azurerm_eventhub" "frontdoor_waf" {
+  name = "${var.resource_prefix}front_door-waf-log"
+  namespace_name = azurerm_eventhub_namespace.eventhub_namespace.name
+  resource_group_name = var.resource_group
+  partition_count = 1
+  message_retention = 1
+}
+
+resource "azurerm_eventhub_namespace_authorization_rule" "frontdoor_auth_rule" {
```

Comment:
We're also going to need an auth rule for Splunk that is listen only. I'm wondering if we keep this module limited to the eventhub_namespace, then we create a separate module that's something like `event_hub_log_stream` that we can reuse within each module requiring logging (like front_door, storage_account, etc), so the event hub module doesn't get complex and repetitive, as we're going to have _a lot_ of logs to capture.

If you want to pull the `front_door` event hub resources, and switch the `front_door` module to use a data reference for the existing event hub, I can handle creating the log stream module in #440, then switch over the front_door to use that.